src/AnimatedRegion.ts

```diff
@@ -82,27 +95,13 @@ export default class AnimatedMapRegion extends AnimatedWithChildren {
 
   private __getValue() {
     return {
-      latitude: this.latitude.__getValue(),
-      longitude: this.longitude.__getValue(),
-      latitudeDelta: this.latitudeDelta.__getValue(),
-      longitudeDelta: this.longitudeDelta.__getValue(),
+      latitude: (this.latitude as any)._value,
+      longitude: (this.longitude as any)._value,
+      latitudeDelta: (this.latitudeDelta as any)._value,
+      longitudeDelta: (this.longitudeDelta as any)._value,
```

Comment:
I have to cast them to any because _value is private on Value and typescript will throw an error